package.json

```diff
@@ -1,13 +1,18 @@
 {
+  "scripts": {
+    "dev": "concurrently 'cd src/main/webapp && npm install && npm run build:dev' 'mvn package appengine:run'",
```

Comment:
These originally did not have `npm install` in them. I patched them.

Reply:
I don't think `npm install` is usually put within scripts. Ideally you only run it periodically as opposed to each time you start the dev server

Reply:
Then perhaps I could add another make target that runs npm install in the webapp project, and then update the top-level build instructions in README.md.